shared/wallets/trustline/asset.tsx

```diff
@@ -53,22 +59,44 @@ const bodyExpanded = (props: Props) => (
   <Kb.Box2 direction="vertical" style={styles.bodyExpanded} fullHeight={true}>
     {getCode(props)}
     {getIssuerVerifiedDomain(props)}
-    <Kb.Text type="BodySmall" style={styles.marginTopXtiny}>
-      Issuer:
-    </Kb.Text>
     <Kb.Text type="BodySmall" lineClamp={2} ellipsizeMode="middle">
       {props.issuerAccountID}
     </Kb.Text>
-    <Kb.Box style={{...Styles.globalStyles.flexBoxRow, flex: 1}} />
-    <Kb.Button
-      mode="Secondary"
-      type="Wallet"
-      small={true}
-      disabled={!props.onOpenInfoUrl}
-      label={props.infoUrlText}
-      style={Styles.collapseStyles([styles.marginTopXtiny, styles.viewDetails])}
-      onClick={stopPropagation(props.onOpenInfoUrl)}
-    />
+    <Kb.ButtonBar direction="row" align="flex-start" small={true}>
+      {!!props.depositButtonText && (
+        <>
```

Comment:
why do we need the fragment here and below?

Reply:
Thanks, weren't needed, removed